### ISSUE TITLE
Linking multiple login options

### ISSUE THREAD

**Issue by bartduisters** [keystonejs/keystone#2285]
# Feature request

## Is your feature request related to a problem? Please describe.

After using FeathersJS I got used to being able to easily link multiple login methods together. This is currently not possible with KeystoneJS.

### Use case

Someone logs in with Google to create an account, an account with a 'googleId' gets created.

While the person is logged in, he goes through the person of logging in with Facebook. Instead of creating a separate account with a 'facebookId' that is not linked to the user at all. The 'facebookId' is added to the already logged in user.

When the user has been logged out and tries to log in, the user can choose either Google or Facebook to log in to the same account.

### In short
Not logged in -> call is made to authenticate with Google/Facebook -> new account is created.
Logged in already -> call is made to authenticate with Google/Facebook -> no new account is created, instead the credentials are added to the already logged in user.

## Additional context

This would work for PasswordStrategy combined with any PassportStrategy and for any PassportStrategy combined with any other PassportStrategy.

**Comment by Yair-Ohana** [comment]
Is there any progress with this issue?
@bartduisters Did you solved it?

**Comment by bartduisters** [comment]
@Yair-Ohana I kept using my own FeathersJS backend for all my projects.

**Comment by bladey** [comment]
**Keystone 5** has officially moved into active maintenance mode as we push towards the next major new version **Keystone Next**, you can find out more information about this transition [here](https://github.com/keystonejs/keystone/issues/4962).

In an effort to sustain the project going forward, we're cleaning up and closing old issues such as this one. If you feel this issue is still relevant for **Keystone Next**, please let us know.
